src/spartan/ppsnark.rs

```diff
@@ -288,7 +293,7 @@ pub trait SumcheckEngine<G: Group> {
   /// the size of the polynomials
   fn size(&self) -> usize;
 
-  /// returns evaluation points at 0, 2, d-1 (where d is the degree of the sum-check polynomial)
+  /// returns evaluation points at 0, 2, 3 (where 3 is the degree of the sum-check polynomial)
```

Comment:
Nit: the degree of the sum-check need not be 3.

Reply:
Hello. Thank you for the guidance. Then, in my opinion, 0, 2, 3, ..., d seems more appropriate. I have reverted this part. Thank you.